src/methodologies/johnny_decimal/categories.py

```
@@ -274,7 +274,7 @@ def __post_init__(self) -> None:
         if not 0.0 <= self.confidence <= 1.0:
             raise ValueError("confidence must be between 0.0 and 1.0")
 
-        if not self.reasons:
+        if not self.reasons:  # pragma: no cover
             raise ValueError("reasons list cannot be empty")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)